packages/cli/src/rpc/appmap/stats.ts

```diff
@@ -25,10 +32,11 @@ export async function appmapStatsHandler(appmapDir: string): Promise<AppMapRpc.S
     }
   );
   await processNamedFiles(appmapDir, 'canonical.sqlTables.json', collectStrings(tables));
-  let numAppMaps = 0;
   await processNamedFiles(appmapDir, 'metadata.json', async () => numAppMaps++);
 
   return {
+    name: config.name,
+    path: config.path,
```

Comment:
Is `path` being removed / not reported? Is `path` actually `directory`? Consider renaming it.